log/experimental_level/level.go

```diff
@@ -147,10 +63,87 @@ func (l *logger) Log(keyvals ...interface{}) error {
 	return l.next.Log(keyvals...)
 }
 
-func makeSet(a []string) map[string]struct{} {
-	m := make(map[string]struct{}, len(a))
-	for _, s := range a {
-		m[s] = struct{}{}
-	}
-	return m
+// Option sets a parameter for the leveled logger.
+type Option func(*logger)
+
+// AllowAll is an alias for AllowDebugAndAbove.
+func AllowAll() Option {
+	return AllowDebugAndAbove()
+}
+
+// AllowDebugAndAbove allows all of the four default log levels.
+func AllowDebugAndAbove() Option {
+	return allowed(levelDebug | levelInfo | levelWarn | levelError)
+}
```

Comment:
Looking at this through the fresh eyes of someone reviewing the Proposal doc, I'm raising my eyebrow a little bit at the AllowFooAndAbove nomenclature, specifically the _AndAbove_ bit, which seems necessarily implicit. Therefore the bitfield implementation here, which allows arbitrary selection of levels, may be needless. What do you think about [this patch](https://gist.github.com/peterbourgon/d8efcb79812e5c03373f1c2a3abd4f56)?